x/ecocredit/client/tx.go

```diff
@@ -172,6 +173,34 @@ Parameters:
 	}
 }
 
+func txCancel() *cobra.Command {
```

Comment:
it doesn't have to be part of this PR but it would be good to add some integration tests for the ecocredit CLI for next release
I believe https://github.com/regen-network/regen-ledger/issues/150 should be added to v1.1 - Baikal Upgrade milestone, wdyt @clevinson @aaronc?

Reply:
Yeah, sounds good - how do we test the CLI currently?

Reply:
see for instance in `x/group/client/testsuite`